### PR TITLE
Toghrulj

### PR DESCRIPTION
**Pull request by toghrulito**
Please check my

### CODE REVIEW

**Review by ZibaMammadzada — APPROVED** [review]
Elachi!

**Comment by ZeenatHaseer** [comment]
Thank you

File: src/test/java/com/NB_CRM/pages/BasePage.java [hunk 1]
```diff
@@ -1,10 +1,79 @@
...
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.FindBys;
 import org.openqa.selenium.support.PageFactory;
 
 public class BasePage {
     public BasePage() {
         PageFactory.initElements(Driver.getDriver(),this);
     }
+    
+
```
Comment (ZeenatHaseer) [inline src/test/java/com/NB_CRM/pages/BasePage.java:14]: Good job

**Review by JavidKK — APPROVED** [review]
geweng

**Review by JavidKK — APPROVED** [review]
Maharram

**Review by JavidKK — COMMENTED** [review]
sa

**Review by Maharram1984 — APPROVED** [review]
Ureyim uzulduyee

**Review by ZeenatHaseer — APPROVED** [review]

**Review by JavidKK — APPROVED** [review]
lk

**Review by JavidKK — APPROVED** [review]

**Pull request closed without merging**